signac/contrib/linked_view.py

```diff
@@ -99,19 +99,26 @@ def create_linked_view(project, prefix=None, job_ids=None, index=None, path=None
         raise RuntimeError(err_msg)
 
     path_function = _make_path_function(jobs, path)
-
-    links = {}
+    incomplete_links = {}
```

Comment:
The old code generated the link mapping in a way that we could not identify.

The old code put the user specified path as the key in the dictionary. Everywhere else we put the actual job path as the key of the dictionary.

I changed the 1 place in the code that was affected by swapping this dictionary around.

I renamed `links` to `incomplete_links` to see both ways of doing things